src/shared/ListRepo/ReposTable/ReposTable.tsx

```diff
@@ -105,7 +105,10 @@ const ReposTable = ({
   })
   const isCurrentUserPartOfOrg = ownerData?.isCurrentUserPartOfOrg
 
-  const { data: tierName } = useTier({ provider, owner })
+  const { data: tierName } = useTier({
+    provider,
+    owner: owner || userData?.user?.username,
```

Comment:
the fallback is the signed in user and `owner` is the org being viewed? Shouldn't this always return the org's plan tier and shouldn't ever return the user's plan tier?